src/interaction-handlers/games/rps.ts

```diff
@@ -18,10 +14,12 @@ export class RpsHandler extends InteractionHandler {
   }
 
   // Get the game info and the interaction type
-  public override parse(interaction: ButtonInteraction): Maybe<{
-    gameId: number;
-    sign: RpsGameSign;
-  }> {
+  public override parse(interaction: ButtonInteraction):
```

Comment:
In Sapphirev4, Maybe no longer exists. We use Option.None and Option.Some instead.